app/src/main/java/com/onboarding/pronosticAR/data/response/WeatherApiResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.onboarding.pronosticAR.data.response
+
+import com.google.gson.annotations.SerializedName
+
+data class WeatherApiResponse(
+    @SerializedName("list")
+    val list: List<WeatherForecastResponse>
```

Comment:
in the Response classes, its a good aproach initilize with a empty value in case that any register cames empty .

Reply:
done

Reply:
here you need to asign a emptyList() ;)